ydb/public/lib/ydb_cli/commands/ydb_dynamic_config.cpp

```diff
@@ -22,6 +22,16 @@ TString WrapYaml(const TString& yaml) {
     return out.Str();
 }
 
+TString WrapStaticConfig(const TString& yaml) {
+    auto doc = NFyaml::TDocument::Parse(yaml);
+    doc.Root().Map().at("config").Insert(doc.Root());
+    doc.Root().Map().at("metadata").Insert(NFyaml::TNodeRef::CreateMap());
```

Comment:
Do not remember method CreateMap, moreover it looks impossible as far as we need "linked" document to create anything